src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/prison/PrisonApiClient.kt

```diff
@@ -27,6 +28,41 @@ class PrisonApiClient(@Qualifier("oauthPrisonClient") val prisonerApiWebClient:
       .onErrorResume { webClientErrorHandler(it) }
   }
 
+  fun hdcStatuses(bookingIds: List<Long>): List<PrisonerHdcStatus> {
```

Comment:
I know you're following the existing example but generally you prefix these with `get`, if you're just retrieving something